kmod/muser.c

```diff
@@ -856,6 +853,31 @@ static int register_notifier(struct mdev_device *const mdev)
 				      &events, &mudev->iommu_notifier);
 }
 
+static int dma_unmap_all(struct muser_dev *mudev)
+{
+	struct vfio_dma_mapping *dma_map;
+	unsigned long length;
+	LIST_HEAD(head);
+
+	mutex_lock(&mudev->dev_lock);
+	while (!list_empty(&mudev->dma_list)) {
```

Comment:
Can we add a TODO to optimize this so time in the future? E.g. use `list_replace`?

Reply:
Done